src/components/hooks/useSpeakers.js

```diff
@@ -0,0 +1,17 @@
+import {useEffect} from 'react';
+
+/**
+ * Custom hook that sets a media element reference with given speaker device id attached to it.
```

Comment:
custom hook that sets the output of an html media element to the specified speaker device